configure

```diff
@@ -28,6 +28,16 @@ export CXXFLAGS
 export CPPFLAGS
 export CFLAGS
 
+if [ -z "${RTESTON}" ] ; then
+    RTESTON=OFF
+fi
+
+if [ ! -z ${USEDISTCC} ] ; then
```

Comment:
Perhaps setting these variables above we could do something like:

CC=${CC:-$(${RCALL} CMD config CC)}

This used bash's "set if empty or undefined syntax. This will allow the user to easily override the R defaults. And simplify this code here too.

What do you think?